### PR TITLE
change default interval to 60s

### PR DESCRIPTION
**Pull request by marcolan018**
open-cluster-management/backlog#12622

Signed-off-by: Marco <llan@redhat.com>

### CODE REVIEW

File: pkg/tests/observability_default_config_test.go [hunk 1]
```diff
@@ -34,7 +34,7 @@ var _ = Describe("Observability:", func() {
 		}
 		observabilityAddonSpec := mcoRes.Object["spec"].(map[string]interface{})["observabilityAddonSpec"].(map[string]interface{})
 		Expect(observabilityAddonSpec["enableMetrics"]).To(Equal(true))
-		Expect(observabilityAddonSpec["interval"]).To(Equal(int64(30)))
+		Expect(observabilityAddonSpec["interval"]).To(Equal(int64(60)))
```
Comment (clyang82) [inline pkg/tests/observability_default_config_test.go:37]: The value is from MCO CR in https://github.com/open-cluster-management/observability-gitops/blob/ac83f3787f1574f568d4665f2a2a111f85ea720b/mco/e2e/v1beta2/observability.yaml#L12. I am not sure if you want to change as well. I think we may just leave as it is - the customized value is 30s.

**Comment by songleo** [comment]
we need to update the doc as well. the default value is 30s in doc.

**Comment by marcolan018** [comment]
@songleo 
I created doc issue https://github.com/open-cluster-management/backlog/issues/12628 for all default values updated

File: pkg/tests/observability_default_config_test.go (hunk 1 above)
Comment (morvencao) [reply pkg/tests/observability_default_config_test.go:37]: We're using `interval: 30` for e2e cases. So we don't need to change this.
We may later need to update this to get the interval from MCO CR.

Comment (marcolan018) [reply pkg/tests/observability_default_config_test.go:37]: @morvencao 
thanks, close this pr

**Pull request closed without merging**